x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/related_integrations/related_integrations.cy.ts

```diff
@@ -5,239 +5,407 @@
+    integration.enabled ? ': enabled' : ''
+  }`.trim();
+}
+
+/**
+ * Agent policy and AWS package policy can be generated by Kibana by opening a page
+ * `app/integrations/detail/aws-1.17.0/overview?integration=cloudfront` then
+ * click the button `Add Amazon CloudFront` -> fill in `Query URL` -> press `Preview API request` at the bottom.
+ *
+ * The contents below is a shrunk copy from the flyout with the only required inputs.
+ */
+const AWS_PACKAGE_POLICY: PackagePolicyWithoutAgentPolicyId = {
+  package: {
+    name: 'aws',
+    version: '1.17.0',
+  },
+  name: 'aws-1',
+  namespace: 'default',
+  inputs: {
+    'cloudtrail-aws-s3': {
+      enabled: false,
+      streams: {
+        'aws.cloudtrail': {
+          enabled: true,
+          vars: {
+            fips_enabled: false,
+            tags: ['forwarded', 'aws-cloudtrail'],
+            preserve_original_event: false,
+            cloudtrail_regex: '/CloudTrail/',
+            cloudtrail_digest_regex: '/CloudTrail-Digest/',
+            cloudtrail_insight_regex: '/CloudTrail-Insight/',
+            max_number_of_messages: 5,
+          },
+        },
+      },
+    },
+    'elb-aws-s3': {
+      enabled: false,
+      streams: {
+        'aws.elb_logs': {
+          enabled: true,
+          vars: {
+            fips_enabled: false,
+            tags: ['forwarded', 'aws-elb-logs'],
+            preserve_original_event: false,
+            max_number_of_messages: 5,
+          },
+        },
+      },
+    },
+    'firewall-aws-s3': {
+      enabled: false,
+      streams: {
+        'aws.firewall_logs': {
+          enabled: true,
+          vars: {
+            fips_enabled: false,
+            tags: ['forwarded', 'aws-firewall-logs'],
+            preserve_original_event: false,
+            max_number_of_messages: 5,
+          },
+        },
+      },
+    },
+    's3-aws-s3': {
+      enabled: false,
+      streams: {
+        'aws.s3access': {
+          enabled: true,
+          vars: {
+            fips_enabled: false,
+            tags: ['forwarded', 'aws-s3access'],
+            preserve_original_event: false,
+            max_number_of_messages: 5,
+          },
+        },
+      },
+    },
+    'waf-aws-s3': {
+      enabled: false,
+      streams: {
+        'aws.waf': {
+          enabled: true,
+          vars: {
+            fips_enabled: false,
+            tags: ['forwarded', 'aws-waf'],
+            preserve_original_event: false,
+            max_number_of_messages: 5,
+          },
+        },
+      },
+    },
+    'cloudfront-aws-s3': {
+      enabled: true,
+      streams: {
+        'aws.cloudfront_logs': {
+          enabled: true,
+          vars: {
+            queue_url: 'https://example.com',
+            fips_enabled: false,
+            tags: ['forwarded', 'aws-cloudfront'],
+            preserve_original_event: false,
+            max_number_of_messages: 5,
+          },
+        },
+      },
+    },
+  },
+};
```

Comment:
Can we move these types, utils and mocks to separate files from the main tests? Probably their own `integrations.ts` files in the `/helpers` and `/tasks` dir?

Reply:
It's related to this test only. Moving it somewhere will reduce test's transparency. It makes sense if there is another test reusing the same policy.